src/utils/i18n.ts

```diff
@@ -25,6 +25,11 @@ i18n
     backend: {
       loadPath: "./assets/locales/{{lng}}/{{ns}}.json",
     },
+
+    react: {
+      useSuspense: (global as any).__DEV__ ? false : true,
```

Comment:
lint gives a warning, but could not find a solution for the global type. Should add a ignore comment:
`// eslint-disable-line` or `// eslint-disable-next-line`.